core/listings.go

```diff
@@ -1219,7 +1219,7 @@ func validatePhysicalListing(listing *pb.Listing) error {
 		for _, region := range shippingOption.Regions {
 			if int(region) == 0 {
 				return errors.New("Shipping region cannot be NA")
-			} else if int(region) > 246 && int(region) != 500 {
+			} else if int(region) > 247 && int(region) != 500 {
```

Comment:
This is painful but okay for now. Tracking https://github.com/OpenBazaar/openbazaar-go/issues/1177 for later.